policy/modules/system/sysnetwork.fc

```diff
@@ -81,13 +81,14 @@ ifdef(`distro_redhat',`
 
 /run/dhclient.*	--	gen_context(system_u:object_r:dhcpc_runtime_t,s0)
 /run/dhcpcd(/.*)?		gen_context(system_u:object_r:dhcpc_runtime_t,s0)
+/run/resolvconf	-d	gen_context(system_u:object_r:net_conf_t,s0)
+/run/resolvconf/.*	gen_context(system_u:object_r:net_conf_t,s0)
```

Comment:
I missed this on the previous version. Are these the only `net_conf_t` files being modified? These should probably be switched to a resolvconf runtime type and then change sysnet_read_config() to call a resolvconf interface that allows read access to these files.

Reply:
Adding a block like:
```m4a
optional_policy(`
    resolvconf_read_runtime($1);
')
```
To `sysnet_read_config()` does not work, since `optional` can not be inside a `tunable`, e.g. `nis_use_ypbind`:
https://github.com/SELinuxProject/refpolicy/blob/7af9eb3e91961e0978adc892ec131edb98bd6c28/policy/modules/services/nis.if#L3-L92

Reply:
Try doing like `nsswitch_domain`, where you create a new attribute in nis.te, apply the rules of `nis_use_ypbind` to the attribute, and then the interface just does a `typeattribute`.